src/app/static/src/app/components/strategisePage.vue

```diff
@@ -63,8 +58,16 @@
                             class="detailsTable"></strategy-table>
           </div>
         </div>
-        <h2 v-else class="h4 text-center">Select a row in the table above to see details of the selected strategy</h2>
+        <h2 v-else class="h4 text-center mt-5">Select a row in the table above to see details of the selected
+          strategy</h2>
       </div>
+      <b-alert :show="errors.length > 0" variant="danger" dismissible @dismissed="dismissErrors">
```

Comment:
Was this just moved as an opportunistic change to improve readability?

Reply:
If left exactly where it was it would have been pulled into `v-if="strategies.length"`, which was moved from `strategies-table` tag to containing `div` in order to accommodate the selected strategy. Could have put it below `loading-spinner` but moving it end seems more logical?